src/inmanta/app.py

```diff
@@ -83,6 +83,7 @@
 
 @command("server", help_msg="Start the inmanta server")
 def start_server(options: argparse.Namespace) -> None:
+    LOGGER.info(f"Starting inmanta-server version {COMPILER_VERSION}")
```

Comment:
The compiler version and the product version are not the same thing. The log line should contains the product version. This is the version of the product python package. For ISO the product package is called `inmanta-service-orchestrator`. For OSS it's called `inmanta`.

Reply:
Ow now I see, thanks